vision/getting-started/image_editing.ipynb

```diff
@@ -0,0 +1,573 @@
+{
```

Comment:
<p>Please use the latest notebook template: <a href="https://github.com/GoogleCloudPlatform/generative-ai/blob/main/notebook_template.ipynb" target="_blank">https://github.com/GoogleCloudPlatform/generative-ai/blob/main/notebook_template.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>